README.md

```diff
@@ -237,61 +239,46 @@ the output to a log file or other log management system.  See the
+The following package parameters can be set:
 
-See the section on [Privileges](#privileges) for information on the
-capabilities the agent requires.
+ * `/access_token` - The token used to send metric data to SignalFx. If not specified, the value in the `\ProgramData\SignalFxAgent\token` file (if it exists) will be used. See the **note** below for details regarding the token and starting the SignalFx Smart Agent service after installation.
+ * `/ingest_url` - Base URL of the SignalFx ingest server. If not specified, the value found in the `\ProgramData\SignalFxAgent\ingest_url` file (if it exists) will be used. Otherwise, defaults to `https://ingest.signalfx.com`.
+ * `/api_url` - Base URL of the SignalFx API server. If not specified, the value found in the `\ProgramData\SignalFxAgent\api_url` file (if it exists) will be used. Otherwise, defaults to `https://api.signalfx.com`.
+ * `/install_dir` - Installation directory. Defaults to `\Program Files\SignalFx\SignalFxAgent`.
 
-3) Run the agent by invoking the agent executable
-`SignalFxAgent\bin\signalfx-agent.exe -config <path to config.yaml>`.  By default,
-the agent logs only to stdout/err. If you want to persist logs, you must direct
-the output to a log file or other log management system.  See the
-[signalfx-agent command](./docs/signalfx-agent.1.man) doc for more information on
-supported command flags.
+The agent will look for a configuration file at
+`\ProgramData\SignalFxAgent\agent.yaml`.  If one does not already exist during
+installaion, a sample config file will be copied in place.
 
-4) You may optionally install the agent as a Windows service by invoking the
-agent executable and specifying a few command line flags.  The examples below
-show how to do install and start the agent as a Windows service.
+**Note:** If the `access_token` parameter is not specified and the
+`\ProgramData\SignalFxAgent\token` file does not exist or is empty, the SignalFx
+Smart Agent service will not automatically start after installation/upgrade.  To
+start the service, update `\ProgramData\SignalFxAgent\token` with a valid
+SignalFx token, and then either restart Windows or run the following Powershell
+command:
 
-- Install Service
+```sh
+& "<install_dir>\bin\signalfx-agent.exe" -service "start"
+```
 
-		PS> SignalFx\SignalFxAgent\bin\signalfx-agent.exe -service "install" -logEvents -config <path to config file>
+See [https://chocolatey.org/packages/signalfx-agent/](https://chocolatey.org/packages/signalfx-agent/)
+for more details.
 
-- Start Service
+### Windows Standalone .zip
+If you are unable to use the installer script or Chocolatey, we offer a .zip that can be deployed to the target host. This bundle is available for download on the [Github Releases Page](https://github.com/signalfx/signalfx-agent/releases).
 
-		PS> SignalFx\SignalFxAgent\bin\signalfx-agent.exe -service "start"
+See [here](./docs/advanced-install-options.md#windows-standalone-zip) for details.
```

Comment:
To learn more, see [Windows standalone zip](./docs/advanced-install-options.md#windows-standalone-zip). Do **NOT** use "here" as link text.